fea-rs/src/compile/tables/var_store.rs

```diff
@@ -0,0 +1,312 @@
+//! Building the ItemVariationStore
+//!
+//! this will probably get moved to write-fonts at some point?
+
+use std::collections::HashMap;
+
+use indexmap::IndexMap;
+use write_fonts::tables::variations::{
+    ItemVariationData, ItemVariationStore, VariationRegion, VariationRegionList,
+};
+
+///
+/// this probably ends up living in write-fonts
```

Comment:
Why don't we just do it in write-fonts to begin with?

If we really think it should move I'd much prefer an issue to a comment

Reply:
This was a note-to-self. The main reason to prefer working on this in fea-rs for now is that it avoids us needing to constantly coordinate three repos while we iterate (if it was in write-fonts we'd either need to be publishing all the time or else be carefully coordinating git dependencies)